Code.ts

```diff
@@ -16,13 +16,19 @@ const REMINDER_MINUTES_BEFORE = 5;
 
 // Exact name of the Google Calendar in which events will be created. Leave blank for default calendar
 const TARGET_CALENDAR_NAME = '';
+
 // =====================
 
+// Function to select under "Choose which function to run" in Triggers
+function run() {
+	main();
+}
```

Comment:
spacing looks a little wonky here

```suggestion
function run() {
  main();
}
```